components/NavigationDrawer/NavigationDrawerListItem/index.tsx

```diff
@@ -1,30 +1,31 @@
 import ListItemText from '@mui/material/ListItemText'
 import ListItemIcon from '@mui/material/ListItemIcon'
 import ListItemButton from '@mui/material/ListItemButton'
+import Link from 'next/link'
 // props for the list
 interface ListProps {
   text: string
   icon?: React.ReactNode
   collapsable?: boolean
+  route: string
 }
 
 function NavigationDrawerListItem({
   collapsable = false,
   icon,
   text,
+  route,
 }: ListProps) {
   return collapsable ? (
-    <div>
-      <ListItemButton>
-        <ListItemIcon>{icon}</ListItemIcon>
-        <ListItemText
-          primary={text}
-          primaryTypographyProps={{ variant: 'body2' }}
-        />
-      </ListItemButton>
-    </div>
-  ) : (
     <ListItemButton>
```

Comment:
this `ListItemButton` would need the same `href` prop, but since the other branch gets rid of this logic, its not really necessary. just make sure to `git pull origin main` after your other pr gets merged :)